rbc/remotejit.py

```diff
@@ -65,6 +104,7 @@ def debug(self):
     def local(self):
         sig = Signature(self.remotejit.local)
         sig.signatures.extend(self.signatures)
+        # sig.signature_templates.update(self.signature_templates)
```

Comment:
Remove

Reply:
Fixed via `assert not self.signature_templates` - `local` is used mostly for debugging and when this is needed in relation to templates then we should add the supporting code here.